### ISSUE TITLE
Tokennize function parameters

### ISSUE THREAD

**Issue by GrayJack** [JuliaEditorSupport/atom-language-julia#180]
Tokennize function parameters/arguments so we can highlight like this python example:

![2019-03-31-004715_grim](https://user-images.githubusercontent.com/17973728/55284522-b32aec80-534e-11e9-8448-26dc80f7b85b.png)

**Comment by CameronBieganek** [comment]
Yes, it would be great if anything inside parentheses and before an equal sign was also tokenized this way, so that it work for both function calls and named tuples. The token name could be something like `variable.parameter` (which is what python uses) or `formal-arg`. Here's a screenshot of some Python code:

<img width="445" alt="Screen Shot 2020-11-15 at 11 11 57 AM" src="https://user-images.githubusercontent.com/8310743/99191934-9e72b680-2735-11eb-9032-14d8973c1ae5.png">

I would like to be able to update my color theme to make formal args brown like in the Python version of my color theme.

So to summarize, this would tokenize all arguments in a function declaration as `formal-arg` and it would also tokenize all keyword argument names in function calls and named tuple "calls". However, to be clear, it would not tokenize variables passed to function calls like `x=1; foo(x)`, since in that case `x` is a variable rather than a formal argument.

**Comment by pfitzseb** [comment]
That does seem doable. I wanted to rework our function arg handling anyways, so that seems like a addition.
